templates/procedures/cp-model-quality.md.tmpl

```diff
@@ -0,0 +1,16 @@
+### Quality of Service 
+
+{{companyShortName}} strives to provide a high quality of service
+to all of its customers. This is accomplished through a security 
+architecture that encompasses all of {{companyShortName}}'s operations
+and provides high data confidentiality, integrity, and availability.
+
+An overview of {{companyShortName}}'s architecture can be found in 
+cp-model-architecture.md. {{companyShortName}} uses a highly scalable
```

Comment:
let's make these links: [Security Architecture](cp-model-architecture.md)

and later in the doc: [Application Service Event Recovery](cp-bcdr-app.md)